### ISSUE TITLE
⌨ Home page very low contrast between text and background colours

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#11982]
## ⌨ Accessibility Issue

Very low contrast between text and background colours.

### WCAG 2.1 Criteria

https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

## 🦋 Expected Behaviour

There should not be very low contrast between text and background colours.

## 📋 Steps to Reproduce

1. Download [WAVE Browser Extensions](https://wave.webaim.org/extension/)
2. Navigate to https://talent.canada.ca/
3. WAVE the page
4. Observe 1 X Very low contrast

## 📸 Screenshot

<img width="1440" alt="Screen Shot 2024-11-15 at 08 55 27" src="https://github.com/user-attachments/assets/b23a0b9a-9d5a-405d-95e2-9cd7ce16a4ab">

## ✅ Acceptance Criteria

- [ ] no low contrast errors on homepage

**Comment by tristan-orourke** [comment]
Looking at the html elements, looks like the background is black (`#000`) and the text is white. Is this really a problem?

**Comment by mnigh** [comment]
> Looking at the html elements, looks like the background is black (`#000`) and the text is white. Is this really a problem?

It is overlap of the white text on the background image that is not strictly black that is the problem.

**Comment by tristan-orourke** [comment]
Waiting for access to the original images.

**Comment by mnigh** [comment]
No longer an issue!

## Screenshot
<img width="1426" height="758" alt="Image" src="https://github.com/user-attachments/assets/65507998-1061-47ab-a0f6-f6b33070f57c" />
